src/flexmeasures_client/client.py

```diff
@@ -97,6 +97,12 @@ async def request(
         - the client polling timed out (as indicated by the client's self.polling_timeout)
         """  # noqa: E501
         url = self.build_url(uri, path=path)
+        url_msg = f"url: {url}"
+        json_msg = f"url: {url}"
+        method_msg = f"method: {method}"
+        logging.debug(url_msg)
```

Comment:
I think we can put the the message formatting inside the logging.debug, directly.